src/opnsense/mvc/app/controllers/OPNsense/Base/ApiControllerBase.php

```diff
@@ -91,7 +91,9 @@ public function initialize()
     {
         // disable view processing
         set_error_handler(array($this, 'APIErrorHandler'));
-        self::getTranslator();
+        if (!isset($this->translator)) {
```

Comment:
It's probably better to change the static getTranslator and move the functionality there, this may have side affects.